content/blog/2025/12/2025-12-19-twim.md

```diff
@@ -0,0 +1,268 @@
+> 
+> The website office hour is on holidays and will return in its regular slot on 8th January! In the meantime, you can still find us in the #matrix.org-website:matrix.org room and of course keep sending PRs! See you again at TWIM next year.
+
+<!--more-->
+
+## Dept of Spec 📜
+
+[Andrew Morgan (anoa) {he/him}](https://matrix.to/#/@andrewm:element.io) reports
+
+> Here's your weekly spec update! The heart of Matrix is the specification - and this is modified by Matrix Spec Change (MSC) proposals. Learn more about how the process works at <https://spec.matrix.org/proposals>.
+> 
+> 
+> ### MSC Status
+> 
+> **New MSCs:**
+> * [MSC4390: Room Blocking API](https://github.com/matrix-org/matrix-spec-proposals/pull/4390)
+> * [MSC4389: Image ordering within packs](https://github.com/matrix-org/matrix-spec-proposals/pull/4389)
+> 
+> **MSCs in Final Comment Period:**
+> * [MSC4356: Recently used emoji](https://github.com/matrix-org/matrix-spec-proposals/pull/4356) (merge)
+> 
+> **Accepted MSCs:**
+> * *No MSCs were accepted this week.*
+> 
+> **Closed MSCs:**
+> * *No MSCs were closed/rejected this week.*
+> 
+> ### Spec Updates
+> 
+> [Matrix v1.17](https://spec.matrix.org/v1.17/) was released this week! Featuring improvements to application services, a flurry of clarifications and finally dropping the old system of user mentions in favour of intentional mentions. It also ships with two highly-requested additions to the spec website itself; a version picker drop-down and a endpoints table of contents!
+> 
+> See [the v1.17 release blog post](https://matrix.org/blog/2025/12/18/matrix-v1.17-release/) for the full changelog.
+> 
+> Otherwise if you'd like something to do to fill the time over the holidays, why not [peruse the list of open proposals](https://spec.matrix.org/proposals/)? 🎄📖
+
+## Dept of Trust & Safety ⚖️
+
+### Introducing policyserv
+
+[Jim](https://matrix.to/#/@mackesque:matrix.org) announces
+
+> Hello from the Trust & Safety team! [Earlier this week](https://matrix.org/blog/2025/12/policyserv/) we revealed that we're open-sourcing the implementation of policy servers we use to protect rooms like TWIM.
+> 
+> Please:
+> 
+> - poke around the code
+> - deploy your own
+> - [sign up to use our policy server to protect your rooms](https://github.com/matrix-org/policyserv-setup-bot?tab=readme-ov-file#usage)
+> - [send us](https://github.com/matrix-org/policyserv/issues) bug reports, feature requests and (ideally) pull requests
+> 
+> Join us in the new #policyserv:matrix.org room for more!
+
+## Dept of Servers 🏢
+  
+### Synapse ([website](https://github.com/element-hq/synapse/))
+
+Synapse is a Matrix homeserver implementation developed by Element.
+
+[Andrew Morgan (anoa) {he/him}](https://matrix.to/#/@andrewm:element.io) reports
+
+> We're pausing our regular releases for the holiday season. Expect them to start back up again in early January with a 1.145.0rc1.
```

Comment:
reject?

Reply:
Hm I don't know, it contains information that the team deemed worthy of sharing 🤷